base/shared/src/main/scala/scalaz/tc/semigroup.scala

```diff
@@ -1,13 +1,18 @@
 package scalaz
-package algebra
+package tc
 
+import java.lang.String
 import scala.language.experimental.macros
 
 trait SemigroupClass[A] {
   def append(a1: A, a2: => A): A
 }
 
-trait SemigroupInstances {}
+object SemigroupClass {
+  implicit val StringSemigroup: Semigroup[String] = instanceOf(new SemigroupClass[String] {
+    def append(a1: String, a2: => String) = a1 + a2
+  })
```

Comment:
this is extraneous?

Reply:
How so?

Reply:
doesn't the `Monoid[String]` instance resolve for this? or have I become mistaken?

Reply:
Good point.